lib/steep/services/signature_help_provider.rb

```diff
@@ -98,6 +133,66 @@ def signature_help_for(node, typing)
 
         [items, index]
       end
+
+      def active_parameter_for(method_type, argument_nodes, last_argument_nodes, node)
+        return unless method_type
+
+        positionals = method_type.type.required_positionals.size + method_type.type.optional_positionals.size + (method_type.type.rest_positionals ? 1 : 0) + method_type.type.trailing_positionals.size
```

Comment:
Note: This patch did not take care of `trailing_positionals` because I don't know about it.